extensions/amp-social-share/1.0/social-share-config.js

```diff
@@ -14,19 +14,23 @@
  * limitations under the License.
  */
 
-import {dict} from '../../../src/utils/object';
-
 /**
  * Get social share configurations by supported type.
- * @param  {string} type
- * @return {!SocialShareConfigDef}
+ * @param  {?string} type
+ * @return {?SocialShareConfigDef}
  */
 export function getSocialConfig(type) {
   return BUILTINS[type];
 }
 
 /**
- * @typedef {!Object<string, !JsonObject|string>}
+ * @typedef {{
+ *   shareEndpont:           string,
```

Comment:
Nit: By the styleguide we just need one space between `:` and type.

Reply:
Thanks, updated throughout all files!